lms/djangoapps/verify_student/admin.py

```diff
@@ -40,10 +38,6 @@ def get_readonly_fields(self, request, obj=None):
             return self.readonly_fields + ('status', 'checkpoint', 'user', 'response', 'error')
         return self.readonly_fields
 
-    def has_delete_permission(self, request, obj=None):
```

Comment:
Note to reviewers: this restriction ended up making it really difficult to reset state locally.  It also resulted in a pretty ugly 403 error in Django admin.  I figured it was easier to just remove this and trust people with access to this table to be careful.